README.md

```diff
@@ -236,3 +236,19 @@ spec:
       type: Ready
     observedGeneration: 1
 ```
+
+## Cleanup
+
+1. Remove the Management object:
+  
+`kubectl delete management.hmc <name> -n hmc-system`
```

Comment:
Currently, we support only management object with the `hmc` name. Our code refers to it. Maybe we should explicitly specify this name here (at least for now)?
`kubectl delete management.hmc hmc -n hmc-system`

I think, we will change this instruction anyway when we start supporting multiple `Management` objects.